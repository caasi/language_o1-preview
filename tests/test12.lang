if "text" == "text" then
    1
else
    0;
