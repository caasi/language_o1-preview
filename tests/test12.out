Error: Invalid operands for binary operation
